recipes-core/packagegroups/packagegroup-ni-internal-deps.bb

```diff
@@ -61,3 +61,9 @@ RDEPENDS:${PN} += "\
 RDEPENDS:${PN} += "\
 	lz4 \
 "
+
+# Required by nisetupscripts for pre-installer testing
+# Contact: Zach Hindes <zach.hindes@ni.com>
```

Comment:
Can you add the owning team name, in addition to this engineer contact?

Reply:
Apparently no owning team ([ref](https://dev.azure.com/ni/DevCentral/_workitems/edit/2459411)).